src/current/cockroachcloud/export-metrics.md

```diff
@@ -338,6 +338,67 @@ A subset of CockroachDB metrics require that you explicitly [enable percentiles]
 Exporting Metrics to Azure Monitor from a CockroachDB {{ site.data.products.dedicated }} cluster is in **[limited access](https://www.cockroachlabs.com/docs/{{ site.current_cloud_version }}/cockroachdb-feature-availability)** and is only available to enrolled organizations. To enroll your organization, contact your Cockroach Labs account team. This feature is subject to change.
 {{site.data.alerts.end}}
 
+Exporting metrics to Azure Monitor is only available on CockroachDB {{ site.data.products.dedicated }} clusters that are hosted on Azure. If your CockroachDB {{ site.data.products.dedicated }} cluster is hosted on AWS, you can export metrics to [AWS CloudWatch]({% link cockroachcloud/export-metrics.md %}?filters=aws-metrics-export). If your CockroachDB {{ site.data.products.dedicated }} cluster is hosted on GCP, you can export metrics to [Datadog]({% link cockroachcloud/export-metrics.md %}?filters=datadog-metrics-export) or [Prometheus]({% link cockroachcloud/export-metrics.md %}?filters=prometheus-metrics-export) instead.
+
+Perform the following steps to enable metrics export from your CockroachDB {{ site.data.products.dedicated }} cluster to Azure Monitor.
+
+1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID that will be used in Step 5:
```

Comment:
```suggestion
1. Find your CockroachDB {{ site.data.products.dedicated }} cluster ID, which will be used in Step 5:
```
It may be difficult to remember to rename the references to other steps. I wonder if it is worth linking to the relevant step instead, then if step 5 becomes step 6, `make linkcheck` will report a broken anchor. There are a few more instances below.